app/models/user.rb

```diff
@@ -1,12 +1,12 @@
 class User < ApplicationRecord
-  has_many :comments
```

Comment:
Migrations are failing as it's pointed out by the previous reviewer. Kindly fix the issue.
![migrate](https://user-images.githubusercontent.com/35504149/162587545-bb86556c-0ec6-437b-b48d-5e7f669c2d0b.png)